ui/pages/desktop-pairing/index.scss

```diff
@@ -3,126 +3,39 @@
   flex-flow: column;
   align-items: center;
   padding: 0 30px 0;
+  max-width: 640px;
 
   &__countdown-timer {
-    background: #f2f3f4;
-    border-radius: 15.5px;
-    padding: 7px 0 7px 0;
-    margin: 0 32px 0 32px;
+    background: var(--color-background-default);
+    border-radius: 16px;
+    padding: 8px 0;
```

Comment:
All of our components in the `component-library/` folder are built on top of the `Box` component and accept all Box props so you could remove this CSS and use 
```
<Text
  className="desktop-pairing__countdown-timer"
  variant={TextVariant.paragraph}
  align={TEXT_ALIGN.CENTER}
  backgroundColor={BackgroundColor.backgroundDefault}
  borderRadius={BorderRadius.XL}
  paddingTop={2}
  paddingBottom={2}
```
```suggestion
```

Reply:
Fixed